examples/utils/customWebWorkersConfig.js

```diff
@@ -14,29 +14,21 @@ function UrlExists (url) {
   return http.status !== 404;
 }
 
-let webWorkerUrl = getBlobUrl('https://unpkg.com/cornerstone-wado-image-loader/dist/cornerstoneWADOImageLoaderWebWorker.min.js');
-let codecsUrl = getBlobUrl('https://unpkg.com/cornerstone-wado-image-loader/dist/cornerstoneWADOImageLoaderCodecs.js');
 let webWorkerTaskPath = 'https://rawgit.com/cornerstonejs/cornerstoneWADOImageLoader/master/examples/customWebWorkerTask/convolveTask.js';
 
 // If running with build completed and DIST folder present
-if (UrlExists('../../dist/cornerstoneWADOImageLoaderWebWorker.min.js')) {
-  webWorkerUrl = '../../dist/cornerstoneWADOImageLoaderWebWorker.min.js';
-  webWorkerTaskPath = '../examples/customWebWorkerTask/convolveTask.js';
-}
-if (UrlExists('../../dist/cornerstoneWADOImageLoaderCodecs.js')) {
-  codecsUrl = '../dist/cornerstoneWADOImageLoaderCodecs.js';
+if (UrlExists('../../dist/cornerstoneWADOImageLoader.min.js')) {
```

Comment:
We should move things around so we can serve these more sanely?

Reply:
Open to suggestions